crates/hyperqueue/src/client/output/quiet.rs

```diff
@@ -75,8 +75,9 @@ impl Output for Quiet {
         task_selector: Option<Selector>,
         output_stream: OutputStream,
         task_header: bool,
+        task_status: Vec<Status>,
```

Comment:
The function signature has gotten quite complex. I think that it would be nicer if the function now only received a list of tasks and the filtering by ID and status was done in `commands::job.rs`.